server/app_engine.py

```diff
@@ -343,26 +269,30 @@ def strip_model_entity(model_entity):
     ]
     for prop in props_to_remove:
         if prop in model_entity:
-            model_entity.pop(prop, None)
+            model_entity.puser_roles(prop, None)
```

Comment:
What's going on here?

Reply:
Oh my.   That must be fat fingers combined with tired eyes.  And I do look through diffs before posting.  Sorry about this one.